src/utils/searchParams.js

```diff
@@ -30,7 +30,7 @@ export const selectSearchParamValue = (search, paramToSelect) =>
  * @param {object} params the parsed searchParams object
  */
 // eslint-disable-next-line import/prefer-default-export
-export const strigifySearchParams = (params) => {
+export const stringifySearchParams = (params) => {
```

Comment:
This is a named export. You probably would want to change the names at all the imports as well.

Reply:
It is updated.